src/defs/clientProfilesRepresentation.ts

```diff
@@ -3,7 +3,8 @@ import ClientProfileRepresentation from './clientProfileRepresentation';
 /**
  * https://www.keycloak.org/docs-api/15.0/rest-api/#_clientprofilesrepresentation
  */
+
```

Comment:
Can you remove this newline so the comment is aligned with the interface? That way it will be picked up as JSDoc.